### DISCUSSION TITLE
delete

### DISCUSSION THREAD

**Discussion by gsxgmpkt9020** [BewlyBewly/BewlyBewly#1211] [Q&A] [answered]
delete

**Comment by hakadao** [comment, marked as answer]
我甚麼都沒做只是免費把自己喜歡而弄的東西放了出來也沒有強制你們收費或其他行為，不喜歡就可以不用
怎麼你們可以把我描述成千古罪人一樣

**Reply by gsxgmpkt9020** [reply]
> 我甚麼都沒做只是免費把自己喜歡而弄的東西放了出來也沒有強制你們收費或其他行為，不喜歡就可以不用 怎麼你們可以把我描述成千古罪人一樣

请见谅，我表达方式可能不妥。对不起。
